src/prefect/logging/handlers.py

```diff
@@ -370,6 +372,9 @@ def close(self) -> None:
 
         return super().close()
 
+    def get_log_size(self, log: Dict[str, Any]) -> int:
+        return len(json.dumps(log))
```

Comment:
I think we want to measure it in bytes, right?

```suggestion
        return len(json.dumps(log).encode())
```

Reply:
I mean, I guess if you want it to be accurate.